### PR TITLE
Polish release note related to workspace idling heuristics

### PR DESCRIPTION
**Pull request by dneary**
Update release notes for 6.19.0 - Polish release note related to workspace idling heuristics

Signed-off-by: Dave Neary <dneary@redhat.com>

### CODE REVIEW

File: release-notes/release-6.19.md [hunk 1]
```diff
@@ -23,14 +23,14 @@ Learn more in our [documentation](https://www.eclipse.org/che/docs/infra-support
 
 ---
 
-## Incorrect warning no longer emitted to the logs when workspace idling is disabled
+## Improvements in workspace idling diagnostics
 
-Eclipse Che improved on diagnostics of workspace activity and handling possible error conditions for
-workspace idling and expiration detection. Unfortunately, this caused an unnecessary warning to be
-emitted to the logs about missing workspace expiry time when workspace idling was disabled.
+Due to an improvement in Eclipse Che diagnostics of workspace activity, and error handling related to
```
Comment (ibuziuk) [inline release-notes/release-6.19.md]: do we need comma after `activity` ?

File: release-notes/release-6.19.md [hunk 2]
```diff
@@ -23,14 +23,14 @@ Learn more in our [documentation](https://www.eclipse.org/che/docs/infra-support
...
 ---
 
-## Incorrect warning no longer emitted to the logs when workspace idling is disabled
+## Improvements in workspace idling diagnostics
 
-Eclipse Che improved on diagnostics of workspace activity and handling possible error conditions for
-workspace idling and expiration detection. Unfortunately, this caused an unnecessary warning to be
-emitted to the logs about missing workspace expiry time when workspace idling was disabled.
+Due to an improvement in Eclipse Che diagnostics of workspace activity, and error handling related to 
+and better handling of possible error conditions, we no longer log an unnecessary warning when
```
Comment (ibuziuk) [inline release-notes/release-6.19.md]: `and error handling related to and better handling of possible error conditions` sounds weird

**Review by ibuziuk — COMMENTED** [review]
Some comments inlined

File: release-notes/release-6.19.md (hunk 1 above)
Comment (dneary) [reply release-notes/release-6.19.md]: Yes, this is the Oxford comma, and it is considered important by some people.

File: release-notes/release-6.19.md (hunk 2 above)
Comment (dneary) [reply release-notes/release-6.19.md]: Yes - this was a mistake.

**Review by ibuziuk — APPROVED** [review]

**Pull request merged**